### DISCUSSION TITLE
SQL logic error: no such table: _logs

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#5511] [Q&A]
I'm getting an odd error while making my command line extension to PocketBase. 

(In case it matters, `pocketbase migrate` has already finished running)

```js
$app.logger().error(`foo`)
```

```bash
ERROR foo
2024/09/11 23:37:48 Failed to write log &{{false 1y84bhptxesjn7b 2024-09-12 06:37:48.439Z 2024-09-12 06:37:48.439Z} map[] foo 8} SQL logic error: no such table: _logs (1)
2024/09/11 23:37:48 Logs delete failed SQL logic error: no such table: _logs (1)
```

**Comment by ganigeorgiev** [comment]
Make sure that the migrate and your own custom command uses the the same `pb_data` directory (if you are not using the default one in both cases make sure to specify the `--dir` flag).

If that doesn't help, try to elaborate a little more on your actual setup by providing a more detailed code sample or some steps to reproduce.
